added changes to GCF-ablation-study (#24)
SpecialeBajs/speciale-rapport: article/Experiment/GCF-ablation-study.tex

        LightGCN-plus-sc     & \underline{0.1031}            & \underline{0.2098}                     & 0.03212                          & 0.06261             & \textbf{0.04679}    & \textbf{0.08175}    \\ \hline
        GCF-sum              & 0.09724                       & 0.1988                                 & 0.03095                          & 0.06446             & 0.04075             & 0.07205             \\ \hline
        GCF-sum-minus-sc     & 0.0956                        & 0.1962                                 & 0.03075                          & 0.0629              & 0.04114             & 0.07261             \\ \hline
        GCF-sum-only-ip      & 0.09843                       & 0.199                                  & 0.02878                          & 0.06065             & 0.04114             & 0.07212             \\ \hline
    \end{tabular}
    \caption{NDCG and Recall of the changed methods.}
    \label{tab:ablation-results}
\end{table*}

\subsubsection{Concatenation and weighted summation}

> Looking at Yelp2020 and Amazon-Book on \autoref{tab:ablation-results} the methods that utilize concatenation as layer combination generally performs worse than the methods that utilize summation.

Reviewer: ```suggestion
Looking at Yelp2020 and Amazon-Book on \autoref{tab:ablation-results} the methods that utilize concatenation as their layer combination method generally perform worse than the methods that utilize summation.
```

> On \autoref{fig:GCF-NDCG-ablation-study} most concatenation methods learn faster than the summation methods, but also is prone to early stopping because it starts to decline in performance.

Reviewer: ```suggestion
On \autoref{fig:GCF-NDCG-ablation-study} most concatenation methods learn faster than the summation methods, but the concatenation methods are also prone to early stopping because they start to decline in performance.
```

> This also happens on \autoref{fig:GCF-NDCG-ablation-study-amazon-book} that use Amazon-book,but some concatenation methods still keeps learning unlike on Yelp2020.

Reviewer: Generelt giver den her sætning ikke så meget mening. Det bruger vel ikke amazon-book. De træner med amazon book eller er performed on amazon-book.
Den sidste del af sætningnen er også lidt pepega

On \autoref{fig:GCF-NDCG-ablation-study-amazon-cell-sport} the methods perform differently compared to the Yelp2020 dataset.

> The summation methods train for a shorter amount of time compared to the Yelp2020 dataset.

Reviewer: ```suggestion
The summation methods train for a lower number of epochs compared to the Yelp2020 dataset.
```

> This could be due to the smaller datasets.

Reviewer: ```suggestion
This could be because amazon cell sport is a smaller dataset than Yelp2020.
```

> Generally it can be seen that GCF and GCF-minus-sc performs better than the other methods used in the Amazon-Cell-Sport dataset.

Reviewer: ```suggestion
Generally it can be seen that GCF and GCF-minus-sc perform better than the other methods in \autoref{fig:GCF-NDCG-ablation-study-amazon-cell-sport}.
```

> LightGCN performs better than the other GCF methods that utilize weighted summation as layer combination.

Reviewer: ```suggestion
LightGCN performs better than the GCF methods that utilize weighted summation as their layer combination method.
```

> However with Amazon-Cell-Sport the results varies less, beside for \textit{gcf-sum-only-ip} that is clearly the worst performing.

Reviewer: ```suggestion
However with Amazon-Cell-Sport the results vary less between the methods, although \textit{gcf-sum-only-ip} is clearly performing worse than the other methods.
```
